qa/tasks/vstart_runner.py

```diff
@@ -964,53 +966,18 @@ def __init__(self):
         # methods to work though.
         self.pools = {}
 
+        self.cephadm = False
+        self.testdir = None
+        self.run_cluster_cmd_prefix = self.run_ceph_w_prefix = \
+            [os.path.join(BIN_PREFIX, "ceph")]
```

Comment:
might want to add a comment noting that these variable are overridden here so that parent class can pick them up.

Reply:
@rishabh-d-dave ^

Reply:
Add a note to both vstart_runner.py and ceph_manager.py